### DISCUSSION TITLE
Support for Yeelight W3 Bulb

### DISCUSSION THREAD

**Discussion by tully-8888** [awawa-dev/HyperHDR#588] [General]
Hello, I am having issues with Yeelight W3 Smart Bulb, I got it working using HyperHDR and Piccap on my LG C2 but actually the bulb is not supported. When I'm configuring the bulb in the webui, I can access Yeelight Wizard, the bulb will quickly change colors but It will remain "Unsupported". Suprinsingly, it works but I don't have access to Yeelight menu for choosing the position, etc. The main problem is that after 2 to 3 minutes of working perfectly, the daemon is stopping with this error: HyperHDR caught signal :SIGSEGV 2023-07-01T00:15:27.146 CORE : <ERROR>        /lib/libc.so.6(+0x29e50) [0xf68a9e50] Segmentation fault
#### Steps to reproduce
Manually add bulb's IP.
Try Yeelight Wizard

#### What is expected?
It should work as expected and not crash.

#### What is actually happening?
Yeelight W3 is not identified and I get: SIGSEGV ->  /lib/libc.so.6(+0x29e50) [0xf68a9e50] Segmentation fault and HyperHDR cannot be accessed even from the web UI.

#### System

None
- Database:        read/write

HyperHDR Server OS: 
- Distribution:   webOS TV 7.3.0
- Architecture:   arm64
- Kernel:         linux (5.4.96-266.mlt4tv.4 (WS: 32))
- Qt Version:     5.15.8
- Browser:        Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/16.2 Safari/605.1.15 

Thanks.

**Comment by awawa-dev** [comment]
Hi
As clearly stated, FULL LOGS ARE MANDATORY ;) especially in cases like this one. Because logs are missing and more importantly: currently I don't have Yeelight W3 to reproduce the problem and debug/implement it myself, webos and this type of lamp is not officially supported by HyperHDR so I don't consider it as a HyperHDR's issue and move it to the discussion section. Maybe some user who has this type of lamp will be able to implement it.
